packages/flagship/android/app/build.gradle

```diff
@@ -160,32 +160,32 @@ configurations.all {
     resolutionStrategy {
         eachDependency { DependencyResolveDetails details ->
             if (details.requested.name == 'android-jsc') {
-                details.useTarget group: details.requested.group, name: 'android-jsc-intl', version: 'r224109'
+                details.useTarget group: details.requested.group, name: 'android-jsc-intl', version: 'r236355'
             }
         }
     }
 }
 
 dependencies {
-    compile 'com.google.android.gms:play-services-base:15.+'
-    compile 'com.google.android.gms:play-services-gcm:15.+'
-    compile 'com.google.android.gms:play-services-maps:15.+'
-    compile 'com.google.android.gms:play-services-wallet:15.+'
-    compile 'com.android.support:appcompat-v7:26.1.0'
-    compile 'com.android.support:support-v4:26.1.0'
+    implementation 'com.google.android.gms:play-services-base:15.+'
+    implementation 'com.google.android.gms:play-services-gcm:15.+'
+    implementation 'com.google.android.gms:play-services-maps:15.+'
+    implementation 'com.google.android.gms:play-services-wallet:15.+'
+    implementation "com.android.support:appcompat-v7:27.1.0"
+    implementation 'com.android.support:support-v4:26.1.0'
```

Comment:
All `com.android.support` libraries should be the same version. Right now there's `27.1.0` above and `26.1.0` here. This should also match the `compileSdkVersion`.